README.md

```diff
@@ -53,7 +53,9 @@ docker run --net=host -d -e CONFIGPROXY_AUTH_TOKEN=$TOKEN \
 Usage: orchestrate.py [OPTIONS]
 
 Options:
-
+  --allow_origin                   Set the Access-Control-Allow-Origin header.
```

Comment:
I'm not sure, I haven't used Tornado a ton tbh. I was just trying to follow the convention used by the other options (e.g. `--pool_size `). 

And I named it that to keep it consistent with what jupyter/ipython uses (`--NotebookApp.allow_origin`)